test/modules/TestResultCoordinator/PartitionReceiverHandler.cs

```diff
@@ -55,14 +61,14 @@ public async Task ProcessEventsAsync(IEnumerable<EventData> events)
                                 DateTime enqueuedtime = GetEnqueuedTime(deviceIdFromEvent.ToString(), moduleIdFromEvent.ToString(), eventData);
 
                                 // TODO: remove hardcoded eventHub string in next line
-                                TestOperationResult result = new TestOperationResult(
-                                    moduleIdFromEvent.ToString() + ".eventHub",
+                                var result = new TestOperationResult(
```

Comment:
What's the benefit of changing this to var over TestOperationResult if we know it is going to be a TestOperationResult and won't change?

Reply:
just for readability to shorten the line. :P